trav/main.py

```diff
@@ -86,14 +108,24 @@ def __getattr__(self, variable):
 
 
 class Travis(metaclass=TravisVariables):
+    TRAVIS_BASE_URL = 'https://api.travis-ci.org/'
+    TRAVIS_BADGE_URL = TRAVIS_BASE_URL + '{group}/{repo}.svg?branch={branch}'
```

Comment:
why did these need to move into the class ?

Reply:
To be able to be used in a the tests.